packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -91,6 +93,13 @@ class DDSFooterComposite extends ModalRenderMixin(HybridRenderMixin(HostListener
    */
   _setLanguage?: (language: string) => void;
 
+  /**
+   * The placeholder for `setLanguage()` Redux action that may be mixed in.
+   *
+   * @internal
+   */
+  _languageCallback?: (e) => void;
```

Comment:
Couldn't find the corresponding Redux action, that made me wonder what this is for...?

Reply:
@IgnacioBecerra Gentle reminder - Any thoughts? Thanks!